### ISSUE TITLE
_local-hook-exec throws error if $IGNORE_GLOBAL_HOOKS does not exist.

### ISSUE THREAD

**Issue by davidasnider** [secretlint/secretlint#138]
**What version of secretlint are you using?**
SecretLint version 2.0

**What did you do? Please include the actual steps causing the issue.**
Install pre-commit hooks per documentation, when commiting a file with secrets, an error from `cat` can be observed

```
exit code 1 received, with output: 'cat: /Users/USERID/IGNORE_GLOBAL_HOOKS: No such file or directory
```

**What did you expect to happen?**
No error displayed, it's a non fatal error

**What actually happened? Please include the actual, raw output from secretlint.**
The below error was displayed:
```
exit code 1 received, with output: 'cat: /Users/USERID/IGNORE_GLOBAL_HOOKS: No such file or directory
```

I've written a fix locally and can supply a PR to resolve.

**Comment by azu** [comment]
> I've written a fix locally and can supply a PR to resolve.

OK. Can you create a Pull Request?
